lms/djangoapps/course_home_api/models.py

```diff
@@ -0,0 +1,25 @@
+"""
+Course home api models file
+"""
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from openedx.core.djangoapps.config_model_utils.models import StackedConfigurationModel
+
+
+class DisableProgressPageStackedConfig(StackedConfigurationModel):
+    """
+    Stacked Config Model for disabling the frontend-app-learning progress page
+    """
+
+    STACKABLE_FIELDS = ('disabled',)
+    # Since this config disables the progress page,
+    # it seemed it would be clearer to use a disabled flag instead of an enabled flag.
+    # The enabled field still exists but is not used or shown in the admin.
+    disabled = models.NullBooleanField(default=None, verbose_name=_("Disabled"))
```

Comment:
Nice - that seems like a good decision. You might also have been able to switch the label of the `enabled` field in the admin form and set the form value to be like "this boolean is reversed" - I believe django lets you do that. But  this works too. Just depends on how much you hate the vestigial `enabled` column.

Reply:
Naw this is fine - only would be confusing if you were looking at db. And I think the comment helps here.